service/surf/apps/filters/utils.py

```diff
@@ -125,6 +127,10 @@ def _fetch_eduterm_translations(external_id):
     dutch = labels[0].get("label_nl", default)
     english = labels[0].get("label_en", default)
 
+    if english['value'] == dutch['value']:
```

Comment:
Kun je deze functie hernoemen naar _fetch_translations

Reply:
Bedoel je de `_translate_with_deepl` functie of `_fetch_eduterm_translations`?

Reply:
Ik bedoelde _fetch_eduterm_translations, omdat het mogelijk een deepl vertaling teruggeeft en niets doet met eduterm. Maar simpelweg _fetch_translations werkt niet, want dan is het lastig om _fetch_edustandaard_translations ook te hernoemen. Daar ging de command hieronder over. Sorry voor de onduidelijkheid. Ik hoop dat dit beter is. Maar misschien moeten we dit even laten, tenzij jij wel een snelle oplossing ziet.